client/app/telecom/pack/slots/hubic/pack-hubic.controller.js

```diff
@@ -1,21 +1,24 @@
-angular.module("managerApp").controller("PackHubicCtrl", function (OvhApiPackXdslHubic, $scope, $stateParams, URLS, $q, $translate, Toast, Poller) {
-    "use strict";
+angular.module("managerApp").controller("PackHubicCtrl", class {
 
-    var self = this;
-    var domainDetailsRoute = "/pack/xdsl/{packName}/hubic/services/{domain}/details";
-    var hubicLoginUrl = "https://hubic.com/home/";
-
-    this.loaders = {
-        services: true
-    };
+    constructor ($q, $scope, $stateParams, $translate, OvhApiPackXdslHubic, Poller, Toast, URLS) {
+        this.$q = $q;
+        this.$scope = $scope;
+        this.$stateParams = $stateParams;
+        this.$translate = $translate;
+        this.OvhApiPackXdslHubic = OvhApiPackXdslHubic;
+        this.Poller = Poller;
+        this.Toast = Toast;
+        this.URLS = URLS;
+    }
 
     /**
      * Get detail of a domain voucher already used
      * @param  {string} domain
      */
-    function getVoucherDetails (domain) {
-        var url = domainDetailsRoute.replace("{packName}", $stateParams.packName).replace("{domain}", domain);
-        return Poller.poll(
+    getVoucherDetails (domain) {
+        let url = `/pack/xdsl/${this.$stateParams.packName}/hubic/services/${domain}/details`;
+
+        return this.Poller.poll(
```

Comment:
Don't forget to kill the polling task on `$onDestroy`.

Reply:
Is is necessary, even if is is destroy on finally event of Polling requests ? (See line https://github.com/ovh-ux/ovh-manager-telecom/pull/681/files#diff-f36c78f367013dff31d6f34b3882a1edR71)